server/resolvers/testPlansResolver.js

```diff
@@ -1,28 +1,9 @@
-const { getTestPlans } = require('../models/services/TestPlanVersionService');
-const retrieveAttributes = require('./helpers/retrieveAttributes');
-const { TEST_PLAN_VERSION_ATTRIBUTES } = require('../models/services/helpers');
+const { getTestPlans } = require('../models/services/TestPlanService');
 
-const testPlans = async (_, __, ___, info) => {
-    const requestedFields =
-        info.fieldNodes[0] &&
-        info.fieldNodes[0].selectionSet.selections.map(
-            selection => selection.name.value
-        );
-    const includeLatest = requestedFields.includes('latestTestPlanVersion');
-    const includeHistoric = requestedFields.includes('testPlanVersions');
-
-    const { attributes: testPlanVersionAttributes } = retrieveAttributes(
-        'latestTestPlanVersion',
-        TEST_PLAN_VERSION_ATTRIBUTES,
-        info,
-        true
-    );
-
-    return getTestPlans({
-        includeLatestTestPlanVersion: includeLatest,
-        includeTestPlanVersions: includeHistoric,
-        testPlanVersionOrder: [['updatedAt', 'desc']],
-        testPlanVersionAttributes
+const testPlans = async () => {
+    const plans = await getTestPlans();
```

Comment:
Just to point out that because the default TEST_PLAN_VERSION_ATTRIBUTES are will be used in the function being called, it will force unexpected columns being queried as well. Such as the `tests` column from `TestPlanVersion`, which we know causes a huge overhead when querying from the database.

Maybe you could try using this with the `retrieveAttributes` function which was previously used here and also used in other resolvers? It attempts to map the raw attributes requested in the GraphQL query, to relevant column names for the table(s), so that only those columns are queried for.